world/main.cpp

```diff
@@ -179,6 +179,8 @@ int main(int argc, char **argv)
 	EQTimeTimer.Start(600000);
 	Timer parcel_prune_timer(86400000);
 	parcel_prune_timer.Start(86400000);
+	Timer expired_suspensions_prune_timer(60000);
+	expired_suspensions_prune_timer.Start(60000);
```

Comment:
Does this need to run every minute ?

Once an hour is fine. 

We'll also need to make sure when we're querying for the account during login we are filtering by suspension date as well. That way suspensions are cleared the moment time passes, not when we null out the field.

Reply:
Removed timer per our discussion.